harvester_e2e_tests/integrations/test_4_vm_backup_restore.py

```diff
@@ -121,6 +121,8 @@ def config_backup_target(api_client, conflict_retries, backup_config, wait_timeo
         f'Failed to update backup target to {backup_type} with {config}\n'
         f"API Status({code}): {data}"
     )
+    # sleeping to allow longhorn secret to be built on Harvester side
+    sleep(5)
```

Comment:
Do we have any way of checking the secret existence? Can we check that in a loop with some sleep?

Reply:
Could we loop through checking the status of the backup target by looping through something like `api_client.settings.backup_target_test_connection`? Will that return true if the secret isn't there?